tests/addTask.test.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @jest-environment jsdom
+ */
+import Tasks from '../modules/TasksList.js';
+
+const tasks = new Tasks();
+const task = tasks.newTask('add a remove function', false);
+
+describe('addTask', () => {
+  test('description value from add Task', () => {
+    const { description } = task;
+    expect(description).toBe('add a remove function');
+  });
+  test('Tasks is not empty', () => {
+    expect(tasks.tasks.length).toBe(1);
+  });
+  test('Check if an item was removed', () => {
+    expect(JSON.stringify(tasks.removeTask(0))).toBe(JSON.stringify([]));
+  });
+});
```

Comment:
- [ ] From the requirements of this project, we are asked to add tests for mock functions that used localStorage, Dom. Please let's add these tests
- [ ] Please let's also make sure that the string stating what's being tested is clear enough. Here, I meant the function for adding and deleting a list item. Don't also forget that the functions we are testing should be **add item** and **delete item**